fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -182,7 +192,14 @@ public synchronized long getIncrement() {
      */
     public void writeFDBLuceneFileReference(@Nonnull final String name, @Nonnull final FDBLuceneFileReference reference) {
         LOGGER.trace("writeFDBLuceneFileReference {}", reference);
-        context.ensureActive().set(metaSubspace.pack(name), reference.getTuple().pack());
+        final FDBStoreTimer timer = context.getTimer();
+        if (timer != null) {
+            long start = System.nanoTime();
+            context.ensureActive().set(metaSubspace.pack(name), reference.getTuple().pack());
+            timer.record(FDBStoreTimer.Events.LUCENE_WRITE_FILE_REFERENCE, System.nanoTime() - start);
```

Comment:
This isn't going to tell you anything useful.  Writes are held purely in the client until the transaction is committed. As such, the time measured for a write will always be almost zero.

Reply:
Removed, thanks